### PR TITLE
Godep: fixed missing dependencies

### PR DESCRIPTION
**Pull request by akolb1**
This fixes the following problems:

Missing dependency on github.com/rakyll/pb
Missing dependency on golang.org/x/sys/unix
Updated speakeasy dependency

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -104,6 +104,10 @@
 			"Rev": "ee2372b58cee877abe07cde670d04d3b3bac5ee6"
 		},
 		{
+			"ImportPath": "github.com/rakyll/pb",
```
Comment (xiang90) [inline Godeps/Godeps.json]: @akolb1 @yichengq Do you know where do we have this dependency? I am not aware of it.

Comment (xiang90) [inline Godeps/Godeps.json]: oh... i think it is the bench tool. but are we building the bench tool in the build script?

**Comment by xiang90** [comment]
LGTM. defer to @yichengq

**Comment by yichengq** [comment]
I don't see where `golang.org/x/sys/unix` is used. Could you point it to me?

Why do we update speakeasy?

**Comment by akolb1** [comment]
github.com/boltdb/bolt/bolt_unix_solaris.go uses golang.org/x/sys/unix
speakeasy was updated to include Solaris support
tools/v3benchmark/main.go uses github.com/rakyll/pb

**Comment by akolb1** [comment]
@yichengq github.com/boltdb/bolt/bolt_unix_solaris.go uses golang.org/x/sys/unix
speakeasy was updated to include Solaris support
tools/v3benchmark/main.go uses github.com/rakyll/pb

**Comment by yichengq** [comment]
That makes sense. LGTM

Thanks for updating these.

**Comment by akolb1** [comment]
@yichengq @xiang90 BTW, I discovered that godep update doesn't quite work. It didn't notice speakeasy update and had some other troubles. I had to remove speakeasy dependency from Godeps.json and then run godep save -r ./...

**Pull request merged**